app/routes.js

```diff
@@ -1,16 +1,26 @@
 const { healthCheck } = require('./controllers/healthCheck');
 const { signUp, signIn, getUsers } = require('./controllers/users');
 const { signUpDto, signInDto, getUsersDto } = require('./dtos/users');
+const { RolesType } = require('./fixtures/roles');
 const { authenticated } = require('./middlewares/auth');
 const { mapSnakeToCamel } = require('./middlewares/mappers');
-const { existsEmail } = require('./middlewares/users');
+const { passArgToUserLocals } = require('./middlewares/passArg');
 const { validationSchema } = require('./middlewares/validationSchema');
 
 exports.init = app => {
   app.get('/health', healthCheck);
 
   // user endpoints
-  app.post('/users', [signUpDto, validationSchema, existsEmail, mapSnakeToCamel], signUp);
+  app.post(
+    '/admin/users',
```

Comment:
It may be that middleware is added that is responsible for validating that the user who wants to create an admin user is admin.